### PR TITLE
fix bug related to argument and output

### PR DESCRIPTION
**Pull request by AprilYUZhang**
Set default hap threshold and default geno threshold #146 #133 #132

### CODE REVIEW

File: src/tinypeel/Peeling/PeelingIO.py [hunk 1]
```diff
@@ -85,40 +85,58 @@ def writeGenotypes(pedigree, genoProbFunc, isSexChrom):
...
+                    writeCalledGenotypes(
+                        pedigree,
+                        genoProbFunc,
+                        isSexChrom,
+                        args.out_file + ".geno_" + str(thresh) + ".txt",
+                        thresh,
+                    )
+        else:
+            thresh = 1 / 3
+            writeCalledGenotypes(
```
Comment (gregorgorjanc) [inline src/tinypeel/Peeling/PeelingIO.py:111]: if args.geno:
         if args.geno_threshold:
             for thresh in args.geno_threshold:
                 if thresh < 1 / 3:
                     thresh = 1 / 3
         else:
             thresh = 1 / 3

         if args.binary_call_file:
             writeBinaryCalledGenotypes(
                pedigree,
                genoProbFunc,
                isSexChrom,
                args.out_file + ".called." + str(thresh),
                thresh,
             )
         else:
            writeCalledGenotypes(
  ...
```

Do you see what I have done here?

File: src/tinypeel/Peeling/PeelingIO.py [hunk 2]
```diff
@@ -85,40 +85,58 @@ def writeGenotypes(pedigree, genoProbFunc, isSexChrom):
...
+                pedigree,
+                genoProbFunc,
+                isSexChrom,
+                args.out_file + ".geno_" + str(thresh) + ".txt",
+                thresh,
+            )
+    if args.hap:
+        if args.hap_threshold:
```
Comment (gregorgorjanc) [inline src/tinypeel/Peeling/PeelingIO.py:119]: @AprilYUZhang As above

**Review by gregorgorjanc — CHANGES_REQUESTED** [review]
@AprilYUZhang thanks for this fix! I have one proposed change and then this is ready to be merged in! Fab;)

File: src/tinypeel/Peeling/PeelingIO.py (hunk 1 above)
Comment (gregorgorjanc) [reply src/tinypeel/Peeling/PeelingIO.py:111]:
```
if args.geno:
 TODO: improve this top part
         treshs = [] 
         if len(args.geno_threshold) > 0:
             for thresh in args.geno_threshold:
                 if thresh < 1 / 3:
                     threshs.append(1 / 3)
         else:
             threshs = 1 / 3

        for tresh in treshs:
           if args.binary_call_file:
               writeBinaryCalledGenotypes(
                  pedigree,
                  genoProbFunc,
                  isSexChrom,
                  args.out_file + ".called." + str(thresh),
                  thresh,
               )
           else:
              writeCalledGenotypes(
  ...
```

**Comment by XingerTang** [comment]
@AprilYUZhang Why this pull request is closed before merging in?

**Comment by AprilYUZhang** [comment]
I completely forgot why, but it should be merged in principle. So I reopened.

**Comment by gregorgorjanc** [comment]
@AprilYUZhang @XingerTang this PR is now openned for a long time. Remind me where we stand with this one?

**Comment by XingerTang** [comment]
@gregorgorjanc This PR is ready to merge. :)

**Review by gregorgorjanc — APPROVED** [review]
LGTM!

**Pull request merged**